packages/blocks/src/components/format-quick-bar/styles.ts

```diff
@@ -48,6 +48,11 @@ export const formatQuickBarStyle = css`
     background-color: var(--affine-border-color);
   }
 
+  .database-button:is([disabled]) {
+    color: rgba(0, 0, 0, 0.2);
+    fill: rgba(0, 0, 0, 0.2);
+  }
```

Comment:
The `format-bar-button` already supports the `disabled` state. Does its style meet the usage?

https://github.com/toeverything/blocksuite/blob/62c3bed3075665ef050d53877311e81af1d9ca58/packages/blocks/src/components/button.ts#L51-L57

Reply:
The `disabled` style does not match the current usage.